src/codeu/chat/client/maingui/LoginPanel.java

```diff
@@ -0,0 +1,72 @@
+package codeu.chat.client.maingui;
+import java.awt.*;
+import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+import javax.swing.*;
```

Comment:
Not a suggestion for this pull request, just a question: have you weighed any alternatives to awt or swing?  AWT and Swing are pretty simple but also pretty old frameworks.  It might be worth taking a day to just look up what else is there.

It also might not be worth changing -- there's only 9 weeks and you want to get a simple UI ready so everyone can focus on the parts of the project they're interested in.  But just curious if you've thought about the question at all.